pkg/cmd/cli/backup/create.go

```diff
@@ -120,6 +122,7 @@ func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.StorageLocation, "storage-location", "", "location in which to store the backup")
 	flags.StringSliceVar(&o.SnapshotLocations, "volume-snapshot-locations", o.SnapshotLocations, "list of locations (at most one per provider) where volume snapshots should be stored")
 	flags.VarP(&o.Selector, "selector", "l", "only back up resources matching this label selector")
+	flags.StringVar(&o.OrderedResources, "ordered-resources", "", "order of resources for specific resource types")
```

Comment:
Fixed.

Reply:
I changed it to the below.  Do you think it clear up?
 flags.StringVar(&o.OrderedResources, "ordered-resources", "", "mapping resource type to order of resources of the resource type.  Resource names are in format 'namespace/resourcename' and separated by commas.  Resource type are separated by semi-colon.  Ex: 'pods=ns1/pod1,ns1/pod2;persistentvolumeclaims=ns1/pvc4,ns1/pvc8'.  Optional.")

Reply:
@phuongatemc That definitely helps, thanks! Some minor tweaks, I think.

I would spell out `Example` at the end.

For the first sentence, I find it a little confusing because of how often "resource" is repeated. Perhaps something like `mapping of Kinds to an ordered list of specific resources of that Kind.`?

Reply:
Fixed.